### PR TITLE
DW-578: Updated danish date formats

### PR DESCRIPTION
**Pull request by jekuaitk**
https://jira.itkdev.dk/browse/DW-578

* Updates danish date formats.

### CODE REVIEW

File: config/sync/core.date_format.short.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@ _core:
 id: short
 label: 'Standard kort dato'
 locked: false
-pattern: 'd. F Y'
+pattern: 'm/d/Y - H:i'
```
Comment (rimi-itk) [inline config/sync/core.date_format.short.yml:10]: `m/d/Y` is an unconventional Danish date format (cf. `langcode: da`).

File: config/sync/language/da/core.date_format.long.yml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 label: 'Standard lang dato'
-pattern: 'l, F j, Y - H:i'
+pattern: 'l, j. F, Y - H:i'
```
Comment (rimi-itk) [inline config/sync/language/da/core.date_format.long.yml:2]: Skip the comma before `Y`. Maybe the comma after `l` should be removed as well.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Some small, but important, linguistic questions and suggestions should be addressed.

File: config/sync/core.date_format.short.yml (hunk 1 above)
Comment (jekuaitk) [reply config/sync/core.date_format.short.yml]: Updated.

File: config/sync/language/da/core.date_format.long.yml (hunk 2 above)
Comment (jekuaitk) [reply config/sync/language/da/core.date_format.long.yml]: Updated.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**